app/src/main/java/com/dessalines/thumbkey/utils/Types.kt

```diff
@@ -40,19 +40,23 @@ sealed class KeyAction {
     class ReplaceLastText(val text: String, val trimCount: Int = 2) : KeyAction()
     class ToggleShiftMode(val enable: Boolean) : KeyAction()
     class ToggleNumericMode(val enable: Boolean) : KeyAction()
+    class ToggleEmojiMode(val enable: Boolean) : KeyAction()
     data object DeleteLastWord : KeyAction()
     data object GotoSettings : KeyAction()
     data object IMECompleteAction : KeyAction()
     data object ToggleCapsLock : KeyAction()
     data object SelectAndCopyAll : KeyAction()
+    data object SelectAll : KeyAction()
```

Comment:
It looks like `SelectAndCopyAll` isn't used anymore, so you can delete it, and the `copy_all` string in strings.xml.